charms/kfp-api/tests/unit/test_operator.py

```diff
@@ -432,21 +434,17 @@ def test_install_with_all_inputs_and_pebble(
         )
         assert pebble_exec_command == f"bash -c '{exec_command}'"
         test_env = pebble_plan_info["services"][KFP_API_SERVICE_NAME]["environment"]
-        # there should be 1 environment variable
-        assert 1 == len(test_env)
         assert "test_model" == test_env["POD_NAMESPACE"]
```

Comment:
not sure if it would make sense to test more of the environment here

Reply:
I was also wondering how and if it's really needed. The env rn may be different in the next release, so I did not bother checking that the number of variables were X. If you have suggestions on how we can test this better, let me know.

Reply:
I have added an assertion in the test case that checks the pebble layer. Kindly review it ^_^